modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -305,8 +321,8 @@ export const splitCellIntoColumns = run(opSplitCellIntoColumns, onCell, resize,
 export const splitCellIntoRows = run(opSplitCellIntoRows, onCell, Fun.noop, Fun.noop, Generators.modification);
 export const eraseColumns = run(opEraseColumns, onCells, resize, prune, Generators.modification);
 export const eraseRows = run(opEraseRows, onCells, Fun.noop, prune, Generators.modification);
-export const makeColumnHeader = run(opMakeColumnHeader, onCell, Fun.noop, Fun.noop, Generators.transform('row', 'th'));
-export const unmakeColumnHeader = run(opUnmakeColumnHeader, onCell, Fun.noop, Fun.noop, Generators.transform(null, 'td'));
+export const makeColumnHeader = run(opMakeColumnHeader, onCells, Fun.noop, Fun.noop, Generators.transform('row', 'th'));
```

Comment:
This is technically a breaking API change, so please be sure to include a major version bump for snooker in `versions.txt`.

Reply:
Also this name is now misleading, as we're operating on multiple columns. So instead of a breaking change, maybe a new function should have been added instead (so as to keep consistent with `insertColumnBefore` vs `insertColumnsBefore`)?

Reply:
`versions.txt` still hasn't been updated to reflect this needs a minor version bump.

- [ ] Add snooker minor version bump to `versions.txt`